war/src/test/java/io/lumeer/engine/rest/CollectionServiceIntegrationTest.java

```diff
@@ -461,33 +464,33 @@ public void testUpdateAccessRights() throws Exception {
       client.close();
    }
 
-   @Test
-   public void testSetAndReadAttributeType() throws Exception {
-      setUpCollections(COLLECTION_SET_AND_READ_ATTRIBUTE_TYPE);
-
-      final Client client = ClientBuilder.newBuilder().build();
-      final String attributeName = "dummyAttribute";
-      final String newType = LumeerConst.Collection.COLLECTION_ATTRIBUTE_TYPE_VALUES.get(0);
-
-      collectionFacade.createCollection(COLLECTION_SET_AND_READ_ATTRIBUTE_TYPE);
-      collectionMetadataFacade.addOrIncrementAttribute(getInternalName(COLLECTION_SET_AND_READ_ATTRIBUTE_TYPE), attributeName);
-
-      // set attribute type
-      Response response = client.target(TARGET_URI).path(PATH_PREFIX + COLLECTION_SET_AND_READ_ATTRIBUTE_TYPE + "/attributes/" + attributeName + "/types/" + newType).request(MediaType.APPLICATION_JSON).buildPut(Entity.entity(null, MediaType.APPLICATION_JSON)).invoke();
-      boolean wasSuccessful = response.readEntity(Boolean.class);
-      assertThat(response.getStatus()).isEqualTo(Response.Status.OK.getStatusCode());
-      assertThat(wasSuccessful).isTrue();
-      response.close();
-
-      // read attribute type
-      Response response2 = client.target(TARGET_URI).path(PATH_PREFIX + COLLECTION_SET_AND_READ_ATTRIBUTE_TYPE + "/attributes/" + attributeName + "/types").request(MediaType.APPLICATION_JSON).buildGet().invoke();
-      String attributeType = response2.readEntity(String.class);
-      assertThat(response2.getStatus()).isEqualTo(Response.Status.OK.getStatusCode());
-      assertThat(attributeType).isEqualTo(newType);
-      response2.close();
-
-      client.close();
-   }
+//   @Test
+//   public void testSetAndReadAttributeType() throws Exception {
```

Comment:
The same applies here. If it is no longer needed, this test method should be removed.